skype/skype-ps/skype/Set-CsHuntGroup.md

```diff
@@ -148,27 +164,45 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -DomainController
-PARAMVALUE: Fqdn
+### -Name
+The Name parameter specifies a unique name for the hunt group.
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases: 
+Applicable: Skype for Business Online
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -OverflowAction
+The OverflowAction parameter designates the action to take if the overflow threshold is reached. The OverflowAction property must be set to one of the following three values: DisconnectWithBusy, Forward, and Voicemail The default value is DisconnectWithBusy.
+
+PARAMVALUE: DisconnectWithBusy | Forward | Voicemail
 
 ```yaml
 Type: Object
 Parameter Sets: (All)
-Aliases: DC
+Aliases: 
 Applicable: Skype for Business Online
 
 Required: False
 Position: Named
-Default value: None
+Default value: DisconnectWithBusy
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -Force
-PARAMVALUE: SwitchParameter
+### -OverflowActionTarget
+The OverflowActionTarget parameter represents the target of the overflow action. For example, if the OverFlowAction is set to Forward, this parameter might be set to a SIP address or a PSTN phone number. This parameter is optional unless the OverflowAction is set to Forward, which requires a SIP address.
```

Comment:
We should not mention target can be PSTN number.

Reply:
Fixed